.gitmodules

```diff
@@ -10,3 +10,9 @@
 [submodule "ext/eigen"]
 	path = ext/eigen
 	url = https://github.com/eigenteam/eigen-git-mirror.git
+[submodule "ext/spdlog"]
+	path = ext/spdlog
+	url = https://github.com/gabime/spdlog.git
```

Comment:
wait but now we're not pointing to the proper spdlog, are we?

Reply:
that's official repo if I am not mistaken